loops.js

```diff
@@ -54,3 +62,10 @@
 // >> 9
 // ==========================================
 
+for ( let i = 0; i < 10; i++ ) {
+    if ( i < 3 ) { console.log( i) };
+
+    if ( i > 2 && i < 6 ) { console.log( ">", i) };
+
+    if ( i > 5 && i < 10 ) { console.log( ">>", i) };
+};
```

Comment:
Prima! 
Je zou eventueel nog de " && i < 10" in regel 70 weg kunnen laten, aangezien er in de for-loop verder geen andere condities gecheckt hoeven te worden.